test/IntegrationTests/SmokeTests.cs

```diff
@@ -295,6 +295,38 @@ public void LogsNoneInstrumentations()
 
         collector.AssertEmpty(5.Seconds());
     }
+
+    [Fact]
+    [Trait("Category", "EndToEnd")]
+    public void TracesNoneInstrumentations()
```

Comment:
Why this test is inside `#if !NETFRAMEWORK // The feature is not supported on .NET Framework`?